mismi-s3/test/Test/Mismi/S3.hs

```diff
@@ -66,3 +75,24 @@ withToken t f = do
   u <- liftIO $ T.pack . U.toString <$> U.nextRandom
   let a = Address b (Key . T.intercalate "/" $ ["mismi", u, unToken t])
   awsBracket_ (pure ()) (listRecursively a >>= mapM_ delete >> delete a) (f a)
+
+
+newAddress :: Token -> AWS Address
```

Comment:
Not sure Token is really that useful here really. I think when I tried, I just scaped it, not sure though. I think the original idea was for debugging, but I don't think it has ever been used and not sure it can be really.